plugins/legacy-plugin-chart-sunburst/src/controlPanel.ts

```diff
@@ -34,7 +34,7 @@ export default {
     {
       label: t('Chart Options'),
       expanded: true,
-      controlSetRows: [['color_scheme', 'label_colors']],
+      controlSetRows: [['color_scheme', 'linear_color_scheme']],
```

Comment:
the `label_colors` control isn't used anywhere in this chart, and actually doesn't work (shows up blank). Interestingly enough it's used in several charts. Deferring looking into this to a later PR.